app/src/interfaces/input-code/input-code.vue

```diff
@@ -103,7 +104,12 @@ const stringValue = computed(() => {
 	if (props.value === null || props.value === undefined) return '';
 
 	if (props.type === 'json' || typeof props.value === 'object') {
-		return JSON.stringify(props.value, null, 4);
+		const valueToStringify =
+			typeof props.value === 'string' && isValidJSON(String(props.value))
+				? parseJSON(String(props.value))
+				: props.value;
+
+		return JSON.stringify(valueToStringify, null, 4);
 	}
 
 	return props.value as string;
```

Comment:
Do we want to check for other valid JSON values too? `boolean`, `number` and `null` are also possibilities here in `props.values`

Reply:
Good catch! Makes perfect sense to me 👍 seems like `null` is also handled to return an empty string in line 104, so `String(props.value)` will then cover `boolean` and `number`. Updated!